app/components/createContainer/createcontainer.html

```diff
@@ -486,7 +487,42 @@
               <!-- !privileged-mode -->
             </form>
           </div>
-          <!-- !tab-runtime -->
+          <!-- !tab-runtime --> 
+          <!-- tab-devices -->
+          <div class="tab-pane" id="devices">
+            <form class="form-horizontal" style="margin-top: 15px;">
```

Comment:
The display/style of elements in the form has been updated recently, could you match the new style? 
Have a look at https://github.com/portainer/portainer/blob/develop/app/components/createContainer/createcontainer.html#L414-L442 for an example.

Please re-use keywords such as "add device" instead of "device". Also replace the minus icon to delete an entry with the trash (see the example above).